src/sonic-config-engine/minigraph.py

```diff
@@ -124,6 +124,7 @@ def parse_dpg(dpg, hname):
 
         ipintfs = child.find(str(QName(ns, "IPInterfaces")))
         intfs = []
+        intfnames = {}
```

Comment:
Actually I'm currently not exposing this. Do you think exposing it might be useful?